libs/js-md4/0.3.0/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# v0.3.0 / 2015-12-28
+
+* Added support for update hash.
+* Added support for bytes array output.
+* Added support for ArrayBuffer output.
+
+# v0.2.0 / 2015-12-01
```

Comment:
Remove all source like these, just keep `package.json`. for example: https://github.com/staticfile/static/pull/340/files